aws_lambda_functions/CloudResourceOrchestration/lambda_function.py

```diff
@@ -0,0 +1,101 @@
+import json
+import boto3
+import jira
+
+ssm_client = boto3.client('ssm', region_name='us-east-1')
```

Comment:
I think that it should be under class, put all the parameters under class

Reply:
yes, right.
Here in the lambda case, we always need to reduce the execution time of lambda. So I thought to write it on top, to reduce the calls.